src/main/java/org/apache/maven/plugins/checkstyle/Violation.java

```diff
@@ -196,10 +196,10 @@ public boolean equals( Object other )
       return false;
     }
     Violation violation = ( Violation ) other;
-    return line.equals( violation.line )
+    return Objects.equals( line, violation.line )
```

Comment:
changes should have unit tests

Reply:
I added a test for the equals method. Could you elaborate on what you specifically want to be tested?

Reply:
Can confirm it is running through this line.
![2021-06-04T22:28:41_0001_screenshot](https://user-images.githubusercontent.com/1413391/120859363-63768a00-c584-11eb-9fda-e46974ccf5d7.png)